scipy/stats/_binomtest.py

```diff
@@ -0,0 +1,258 @@
+from collections import namedtuple
+from math import sqrt
+import numpy as np
+from scipy.optimize import brentq
+from scipy.special import ndtri
+from ._discrete_distns import binom
+
+
+ConfidenceInterval = namedtuple('ConfidenceInterval', ['low', 'high'])
```

Comment:
Any chance that making this a tuple will come to bite us?

Reply:
Is the concern that, just as we're now in the process of converting  the return value of the statistics tests from a tuple to an object, we might end up wanting the same conversion for the confidence interval calculations?  A confidence interval really is just two numbers, and I don't anticipate expanding the return value of the confidence interval methods of the various statistical test result objects to return anything else, so I think we're OK with a tuple.   But I wouldn't bet my life on there being *no* chance that we might want more output results from confidence interval calculation someday, and a simple object with `low` and `high` attributes could also work.  What do you think?

Reply:
I don't have a specific objection; it's just something to be considered whenever we see tuples as outputs in `stats`, I think.